samples/tests/test_get_operation.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import os
+
+from samples import get_operation
+
+
+def test_get_operation():
+    project_id = os.getenv('PROJECT_ID', '')
+    location = os.getenv('LOCATION', '')
+    operation_id = os.getenv('OPERATION_ID', '')
+    assert project_id
+    assert location
+    assert operation_id
+
+    get_operation.get_operation(project_id, location, operation_id)
```

Comment:
Do you want to test for what this returns?

Reply:
I ran this test after switching to using fstring and it passed :)

Reply:
I realized I misunderstood your question! I think you meant that we should check if the operation name from the response matches an expected operation name. If that's the case, I just added an assert statement to the test to do so.